app/Models/City.php

```diff
@@ -7,8 +7,22 @@ class City extends Model {
 
     protected $table = 'cities';
 
-    protected $fillable = ['name'];
-//Get all cities order by name
+    protected $fillable = ['id', 'name'];
+
+    static function factoryCity()
```

Comment:
Is it code should be right there? Or put him in factory? Like user factory or resume factory